inductiva/fluids/scenarios/wind_terrain/wind_terrain.py

```diff
@@ -0,0 +1,205 @@
+
+    valid_simulators = [fluids.simulators.OpenFOAM]
+
+    def __init__(self,
+                 terrain: inductiva.world.Terrain,
+                 wind_velocity: List[float],
+                 wind_position: Optional[List[float]] = None,
+                 athmosphere_height: float = 300):
+        """Initializes the `WindTerrain` conditions.
+
+        Args:
+            wind_velocity (List): Velocity of the air flow (m/s).
+            wind_position (List): Position of the wind flow (m) above the height
+            of the terrain.
+            terrain (inductiva.world.Terrain): Terrain object that describes
+                the profile of the terrain with a mesh.
+            athmosphere_height (float): Altitude above the terrain (m) to establish an
+                atmosphere. Notice that the wind_position needs to be inside the
+                atmosphere region.
+        """
+
+        self.wind_velocity = np.array(wind_velocity)
+        # Compute the wind_direction from the velocity vector.
+        self.wind_direction = self.wind_velocity / np.linalg.norm(
+            self.wind_velocity)
+        if wind_position is None:
+            wind_position = [
+                terrain.center["x"], terrain.center["y"],
+                terrain.bounds["z"][1] + 10
+            ]
+
+        self.wind_position = wind_position
+        self.terrain = terrain
+
+        self.top_boundary_height = terrain.bounds["z"][0] + athmosphere_height
+
+        if terrain.bounds["z"][1] >= self.top_boundary_height:
+            raise ValueError(
+                "The terrain height surpasses the top athmosphere boundary.")
+
+    def simulate(
+        self,
+        simulator: inductiva.simulation.Simulator = fluids.simulators.OpenFOAM(
+        ),
+        resource_pool_id: Optional[uuid.UUID] = None,
+        run_async: bool = False,
+        n_cores: int = 1,
+        num_iterations: int = 100,
+    ) -> inductiva.tasks.Task:
+        """Simulates the wind tunnel scenario synchronously.
+
+        Args:
+            simulator: Simulator used to simulate the scenario.
+                Valid simulators: OpenFOAM.
+            num_iterations: Number of iterations to run the simulation.
+            n_cores: Number of cores to use for the simulation.
+            resource_pool_id: Id of the resource pool to use for the simulation.
+                TODO: Change resource pool id to machine group.
+            run_async: Whether to run the simulation asynchronously.
+        """
+
+        self.num_iterations = num_iterations
+        self.n_cores = n_cores
+
+        commands = self.get_commands()
+
+        task = super().simulate(simulator,
+                                resource_pool_id=resource_pool_id,
+                                run_async=run_async,
+                                n_cores=n_cores,
+                                commands=commands)
+
+        task.set_output_class(
+            inductiva.fluids.post_processing.SteadyStateOutput)
```

Comment:
What is this output class?

Reply:
To be added later on, I will remove it from this PR.